tests/test/ExceptionClassNamingSniffNotValid.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Example;
+
+use My\Throwable;
+
+abstract class AbstractWithoutSuffix extends FooException implements \Throwable
+{
+}
+
+abstract class ExtendsNoException extends Nothing
+{
+}
+
+abstract class InheritsNothingException
+{
+}
+
+final class NotAbstractException extends My\Exception\FooException implements \Throwable
+{
+}
+
+class NotFinal extends \Exception
+{
+}
+
+final class NoExtension implements \Throwable
+{
+}
+
+final class StillNoExtension implements FooException
+{
+}
```

Comment:
Some of these files don't contain a trailing EOL. It's either missing from the sniff or we don't properly check tests. Either way, there should be a consistent rule on this.

Reply:
These tests are only for the new exception sniffs to prevent failing tests on new additions in future PRs.

Reply:
I mean the trailing newlines - this is part of PSR-2 and should be fixed here as well.